content/spin/v2/rust-components.md

```diff
@@ -210,25 +209,20 @@ inserts a custom header into the response before returning:
 ```rust
 use anyhow::Result;
 use spin_sdk::{
-    http::{IntoResponse, Request},
+    http::{IntoResponse, Request, Method, Response},
     http_component,
 };
 
 #[http_component]
 async fn send_outbound(_req: Request) -> Result<impl IntoResponse> {
     // Create the outbound request object
-    let req = http::Request::builder()
-        .method("GET")
+    let req = Request::builder()
+        .method(Method::Get)
         .uri("https://random-data-api.fermyon.app/animals/json")
-        .body(())?;
+        .build();
 
     // Send the request and await the response
-    let mut res: http::Response<()> = spin_sdk::http::send(req).await?;
-
-    // Demonstrate modifying the response before passing it
-    // back to the client
-    res.headers_mut()
```

Comment:
Unfortunately, this is not currently possible with the SDK types. I will add a PR to Spin to add this functionality.

Reply:
I take that back - this is possible but slightly awkward:

```
let res = res.into_builder().header("spin-component", "get-animal-fact").build();
```

Shall we include this in the doc or wait for a better solution from the SDK?